Automattic/jetpack
# Instant Search: ensure box-shadow and text-shadow aren't applied to search buttons

Reported by @pablohoneyhoney in p1629911358066600-slack-jetpack-search-team. 

Internal users can see the original issue at p5dIxE-17k-p2.

#### Changes proposed in this Pull Request:

* Ensure box-shadow and text-shadow aren't applied to search buttons. If the underlying theme applies these globally, it can look pretty unsightly:

<img width="487" alt="Screen Shot 2021-08-26 at 14 04 07" src="https://user-images.githubusercontent.com/17325/130888708-2395c168-1546-41ca-a717-8a64be4c1376.png">

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
On a test with Instant Search enabled, add the following Custom CSS via the Customizer:

```
button, html input[type="button"], input[type="reset"], input[type="submit"] {
    box-shadow: inset 0 1px 0 rgba(255,255,255,.5),inset 0 15px 17px rgba(255,255,255,.5),inset 0 -5px 12px rgba(0,0,0,.05);
    text-shadow: 0 1px 0 rgba(255,255,255,.8);
}
```

Trigger the instant search modal and ensure that the buttons do not have any shadow:

<img width="254" alt="Screen Shot 2021-08-26 at 14 04 15" src="https://user-images.githubusercontent.com/17325/130888790-be163683-b596-4812-835d-82068fae7f49.png">
